examples/cfq/models.py

```diff
@@ -126,10 +126,10 @@ def decode_step_fn(carry, x):
       attention = mlp_attention(dec_prev_state, projected_keys,
                                 encoder_hidden_states, attention_mask)
       lstm_input = jnp.concatenate([x,attention], axis=-1)
-      lstm_state, y = lstm_cell(lstm_state, lstm_input)
-      inner_proj_input = jnp.concatenate([x, attention, y], axis=-1)
-      inner_proj_output = inner_projection(inner_proj_input)
-      logits = output_projection(inner_proj_output)
+      lstm_state, h = lstm_cell(lstm_state, lstm_input)
```

Comment:
super nit. you could rename the lstm_state just to state but do not make effort. just go one.